### ISSUE TITLE
The changes:diff: for file.manage appears out of order in 2018.3.0 ?

### ISSUE THREAD

**Issue by zerthimon** [saltstack/salt#48045]
Is this a known problem ?
```
     Changes:   
              ----------
              diff:
                  --- 
                  +++ 
                  @@ -4,11 +4,11 @@
                   # THIS FILE DOES NOT APPLY TO SYSTEMD
                   #
                   #   Please see the documentation for "systemd drop-ins":
                  -#   https://docs.docker.com/engine/admin/systemd/
                  +#   https://docs.docker.com/engine/articles/systemd/
                   #
                   
                   # Customize location of Docker binary (especially for development testing).
                  -#DOCKERD="/usr/local/bin/dockerd"
                  +#DOCKER="/usr/local/bin/docker"
                   
                   # Use DOCKER_OPTS to modify the daemon startup options.
                   #DOCKER_OPTS="--dns 8.8.8.8 --dns 8.8.4.4"
                  @@ -17,4 +17,6 @@
                   #export http_proxy="http://127.0.0.1:3128/"
                   
                   # This is also a handy place to tweak where Docker's temporary files go.
                  -#export DOCKER_TMPDIR="/mnt/bigdrive/docker-tmp"
                  +#export TMPDIR="/mnt/bigdrive/docker-tmp"
                  +
                  +DOCKER_OPTS="$DOCKER_OPTS --storage-driver=overlay --userland-proxy=false"
```
Lines with - should be with + and vice versa ?

**Comment by zerthimon** [comment]
This only appears when running with `test=True`, works correctly with the real run.

**Comment by babilen** [comment]
We've just ran into the same problem on 2018.3.0

**Comment by garethgreenaway** [comment]
This is a duplicate of https://github.com/saltstack/salt/issues/46931 and will be fixed in 2018.3.1.
Thanks!
